packages/build-info/src/metrics.ts

```diff
@@ -9,9 +9,10 @@ export function report(
     context?: 'string'
     severity?: 'info' | 'warning' | 'error'
     metadata?: Record<string, Record<string, any>>
+    client?: Client
   } = {},
 ) {
-  Bugsnag.notify(error, (event) => {
+  ;(options.client || Bugsnag).notify(error, (event) => {
```

Comment:
is the ; a typo?

Reply:
That is the downside of not using semicolons.

Reply:
TIL